theme/static/js/hs_resource_table_community.js

```diff
@@ -0,0 +1,373 @@
+/**
```

Comment:
Most of this script is duplicated from https://github.com/hydroshare/hydroshare/blob/develop/theme/static/js/hs_resource_table.js

I suggest reusing as much as possible from the original script and then writing any new needed functionality in this script.

Reply:
I want to refactor these scripts I just didn't want to take that work on as a part of this PR. Having yet another layer of includes/chaining in this area seems really bad. This eases debugging and gives freedom to refactor one side at a time (since groups is using a different resources query than communities).